app/src/main/java/de/tum/in/tca/ticketcheck/component/ticket/TicketsController.java

```diff
@@ -31,7 +31,6 @@
     public TicketsController(Context context) {
         this.context = context;
 
-        Utils.setSetting(context, Const.CHAT_MEMBER, new ChatMember("ga38fir"));
```

Comment:
@schulzalexander Was this for testing purposes?

Reply:
yes it was^^